scripts/promos.js

```diff
@@ -217,6 +221,11 @@ let fPromoIntervalStarted = false;
  * @param {boolean} fAddRandomness - Whether to append Randomness to the code
  */
 export async function createPromoCode(strCode, nAmount, fAddRandomness = true) {
+    // Ensure the code doesn't have any weird potential-XSS characters
+    if (strCode !== sanitizeHTML(strCode)) {
+        return createAlert('warning', 'Invalid code name!', 2000);
+    }
```

Comment:
This is fine to prevent future xss, but we should also be sanitizing codes just in case.
If we have a code in 2.0 that contains html it will still be displayed as html in 2.0.2/2.1